app/src/main/kotlin/com/intive/picover/validators/TextValidator.kt

```diff
@@ -6,27 +6,26 @@ import com.intive.picover.validators.ValidationStatus.TooLongText
 import com.intive.picover.validators.ValidationStatus.ValidText
 
 class TextValidator private constructor(
-	private val text: String,
 	private val allowEmpty: Boolean,
 	private val allowBlank: Boolean,
 	private val maxLength: Int?,
 ) {
 
-	fun validatingText() = when {
+	fun validatingText(text: String) = when {
```

Comment:
minor: `validatingText` -> `validate`

Reply:
I know :D. It will be renamed in another pull request, but thanks for catching it!